src/PresentationalComponents/Notification/Notification.js

```diff
@@ -0,0 +1,154 @@
+import YourInformation from '../shared/YourInformation';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import './notification.scss';
+import { Main } from '@redhat-cloud-services/frontend-components/Main';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+import { useParams } from 'react-router-dom';
+import {
+  Bullseye,
+  Card,
+  CardBody,
+  CardHeader,
+  Spinner,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { componentMapper } from '@data-driven-forms/pf4-component-mapper/component-mapper/component-mapper';
+import {
+  DATA_LIST,
+  DataListLayout,
+  DESCRIPTIVE_CHECKBOX,
+  DescriptiveCheckbox,
+  Loader,
+  LOADER,
+} from '../../SmartComponents/FormComponents';
+import { FormTemplate } from '@data-driven-forms/pf4-component-mapper';
```

Comment:
`import FormTemplate from '@data-driven-forms/pf4-component-mapper/form-template';`

Reply:
you should specify the exact path

Reply:
Updated, thanks!